src/tests/dbus-tests/test_20_LVM.py

```diff
@@ -606,3 +615,209 @@ def test_resize_physical(self):
+        options['tear-down'] = dbus.Boolean(True)
+
+        crypttab_items = dbus.Dictionary({'name': self.str_to_ay(vgname),
+                                          'options': self.str_to_ay('verify,discard'),
+                                          'passphrase-contents': self.str_to_ay(self.PASSPHRASE),
+                                          'track-parents': True},
+                                          signature=dbus.Signature('sv'))
+        fstab_items = dbus.Dictionary({'dir': self.str_to_ay(vgname),
+                                       'type': self.str_to_ay('ext4'),
+                                       'opts': self.str_to_ay('defaults'),
+                                       'freq': 0, 'passno': 0,
+                                       'track-parents': True},
+                                      signature=dbus.Signature('sv'))
+        options['config-items'] = dbus.Array([('crypttab', crypttab_items),
+                                              ('fstab', fstab_items)])
+
+        self.lv_block.Format('ext4', options, dbus_interface=self.iface_prefix + '.Block')
+        self.addCleanup(self._remove_luks, self.lv_block, vgname)
+        self.luks_uuid = self.get_property_raw(self.lv_block, '.Block', 'IdUUID')
+
+        luks_block_path = self.get_property(self.lv_block, '.Encrypted', 'CleartextDevice')
+        self.luks_block = self.get_object(luks_block_path.value)
+        self.assertIsNotNone(self.luks_block)
+        self.fs_uuid = self.get_property_raw(self.luks_block, '.Block', 'IdUUID')
+
+        # check for present crypttab configuration item
+        conf = self.get_property(self.lv_block, '.Block', 'Configuration')
+        conf.assertTrue()
+        self.assertEqual(conf.value[0][0], 'crypttab')
+
+        # check for present fstab configuration item on a cleartext block device
+        conf = self.get_property(self.luks_block, '.Block', 'Configuration')
+        conf.assertTrue()
+        self.assertEqual(conf.value[0][0], 'fstab')
+
+        child_conf = self.get_property(self.lv_block, '.Encrypted', 'ChildConfiguration')
+        child_conf.assertTrue()
+        self.assertEqual(child_conf.value[0][0], 'fstab')
+        self.assertEqual(child_conf.value, conf.value)
+
+        # check that fstab and crypttab records have been added
+        crypttab = self.read_file('/etc/crypttab')
+        self.assertIn(vgname, crypttab)
+        self.assertIn(self.luks_uuid, crypttab)
+        fstab = self.read_file('/etc/fstab')
+        self.assertIn(vgname, fstab)
+        self.assertIn(self.fs_uuid, fstab)
+
+
+    @udiskstestcase.tag_test(udiskstestcase.TestTags.UNSAFE)
+    def test_teardown_active_vg_unlocked(self):
+        ''' Test tear-down by removing the base VG (not deactivated, unlocked) '''
+
+        name = 'udisks_test_teardown_active_vg_unlocked'
+
+        self._init_stack(name)
+
+#        self.lv_block.Lock(self.no_options, dbus_interface=self.iface_prefix + '.Encrypted')
```

Comment:
Please either remove these, or add an explanation here if this is here because of #781